src/algebra/category/Module/basic.lean

```diff
@@ -55,7 +55,10 @@ universes v u
 
 variables (R : Type u) [ring R]
 
-/-- The category of R-modules and their morphisms. -/
+/-- The category of R-modules and their morphisms. Note that in the case of `R = ℤ`, we can not
+impose here that the `ℤ`-multiplication field from the module structure is defeq to the one coming
+from the `is_add_comm_group` structure (contrary to what we do for all module structures in
+mathlib), which creates some difficulties down the road. -/
```

Comment:
I guess one solution would be to expand this definition (not in this PR) into a structure that joins all the fields of `add_comm_group` and `module`, and then manually create instances of `add_comm_group` and `module R` for the `coe_to_sort` of a `Module`.

Reply:
Yes, in the bundled situation of category theory this would certainly be possible. Note that this creates only one small problem in one specific proof (for now), so I am not sure it is worth the trouble.

Reply:
Yup, so let's leave it like this.